lib/ship_quiet_logistics/documents/shipment_order.rb

```diff
@@ -1,95 +1,111 @@
 module ShipQuietLogistics
   module Documents
     class ShipmentOrder
-      attr_reader :shipment_number, :order, :shipment, :name, :unit
+      attr_reader :config, :shipment_number, :shipment, :name
 
       def initialize(shipment, config)
         @config          = config
         @shipment        = shipment
-        @shipment_number = shipment['id']
+        @shipment_number = shipment.number
         @name            = "#{@config['business_unit']}_ShipmentOrder_#{@shipment_number}_#{date_stamp}.xml"
       end
 
+      def message
+        "Succesfully Sent Shipment #{shipment_number} to Quiet Logistics"
+      end
+
       def to_xml
         builder = Nokogiri::XML::Builder.new do |xml|
           xml.ShipOrderDocument('xmlns' => 'http://schemas.quietlogistics.com/V2/ShipmentOrder.xsd') {
 
-            xml.ClientID @config['client_id']
-            xml.BusinessUnit @config['business_unit']
+            xml.ClientID config['client_id']
+            xml.BusinessUnit config['business_unit']
 
-            xml.OrderHeader('OrderNumber' => @shipment_number,
-                            'OrderType'   => @shipment['order_type'],
-                            'OrderDate'   => DateTime.now.iso8601) {
+            xml.OrderHeader('OrderNumber' => shipment_number,
+                            'OrderType'   => order_type,
+                            'OrderDate'   => shipment.created_at.utc.iso8601) {
```

Comment:
I'm just slightly uneasy at the method-chaining here. Do you worry at all about Spree/Solidus changing their API, and us having to maintain `spree-2-4`, `spree-3-0`, `solidus-1-0`, etc. branches? Not in the timeframe of this PR, but what do you think of adding some kind of adapters between the Spree/Solidus domain objects and the documents?